packages/dialtone-vue3/.storybook/preview.jsx

```diff
@@ -19,6 +20,7 @@ setCustomEmojiJson(customEmojiJson);
 setup((app) => {
   app.use(fixDefaultSlot)
   app.use(DtTooltipDirective);
+  faker.seed(6687422389464139);
```

Comment:
faker global seed, to make sure results are reproducible on percy and don't change on every reload too.

Reply:
I would add this comment on the top of the `faker.seed`